staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go

```diff
@@ -260,7 +260,6 @@ func Run(g *Generator) {
 	if len(g.Conditional) > 0 {
 		fmt.Fprintf(buf, "// +build %s\n\n", g.Conditional)
 	}
-	buf.Write(boilerplate)
```

Comment:
is this related to the package path change, or did this issue exist at HEAD as well? if this issue already exists at HEAD, let's resolve it in a separate PR

Reply:
It doesn't exist at HEAD. If you apply this to HEAD, the boilerplate will be missing when running code generation. So its likely that the changes I have made somewhat lead to this strange behaviour of applying the boilerplate multiple times. 

I need to understand the go-to-protobuf generator a little bit more to fully understand this. Especially where it gets the files from that are "rewritten" is the code above (i.e. if these files already contain the boilerplate or not).

The strange thing is that I have just changed the single line that prints the package path in the go-to-protobuf generator. I will investigate the further within the next days. Lets wait with merging.

Reply:
Happened to me too in https://github.com/kubernetes/kubernetes/pull/106630#issuecomment-1005817409, no clue why...